data/songseed.json

```diff
@@ -1,66 +1,82 @@
 [
```

Comment:
I added two additional songs, but most of these lines are just showing up in the diff because the indentation was changed automatically... so please ignore and sorry!